code/modules/client/loadout/loadout_uniform.dm

```diff
@@ -187,6 +187,34 @@
 	display_name = "sailor outfit"
 	path = /obj/item/clothing/under/costume/sailor
 
+/datum/gear/uniform/misc/perfomer_blue
+	display_name = "blue performer's outfit"
+	description = "oo-ee-oo"
```

Comment:
Not sure what the reference here or on the other blue performer's outfit is

Reply:
They're Miku and Luka's outfits, (Vocaloid)